### ISSUE TITLE
"Error downloading files" with  autodl-irssi.

### ISSUE THREAD

**Issue by bluasoar** [horjulf/docker-rutorrent-autodl#13]
I have been running binhex-rtorrentvpn on unRAID for while now and wanted to implement autodl-irssi. I grabbed the Docker and set everything up I can view the UI for the autodl-irssi and interact with the plugin with no problem but when I try and connect my main rTorrent instance I get an error saying "Make sure autodl-irssi is started and configured properly (eg. password, port number): Error getting files listing: Error: Could not connect: (111) Connection refused" I have checked the conf.php for the plugin to make sure the GUI pass and login are matching the autodl.cfg tried reinstalling with no success. If anyone can point me in the right direction I would appreciate it. If any more info is needed please let me know. Thank you!

**Comment by BrandonG777** [comment]
you aren't pulling in old appdata from your binhex-rtorrentvpn container are you? If so, maybe try to blowing that out and then pulling in rtorrent data as needed. it's been awhile since i set mine up but i dont think much has changed since then.

**Comment by bluasoar** [comment]
> 
> 
> you aren't pulling in old appdata from your binhex-rtorrentvpn container are you? If so, maybe try to blowing that out and then pulling in rtorrent data as needed. it's been awhile since i set mine up but i dont think much has changed since then.

Not entirely sure what you mean by pulling the appdata from the binhex container? The only paths the Autodl-irssi has access to is /config which is separate from the Binhex container and /downloads. I honestly have been trying to set up autodl-irssi for awhile now with the Binhex container with no luck at all. I have to be doing something wrong. Thanks for the reply!

**Comment by BrandonG777** [comment]
well when I started using this container I was using the original linuxserver.io version. so I migrated my appdata (usually everything in the /config folder) over from that and everything worked great. I have since tried to migrate to some other rutorrent containers and haven't had much luck. Even just trying to copy files and folders over individually I haven't had much luck because there seems to be some major differences between rutorrent/rtorrent config files from version to version. Anyway, what I'm getting at is just make sure you're starting with clean appdata folder because you might be running into a conflict somewhere.  Also, I'm not sure if this project is still active, since there has been commits to the original linuxserver.io project that aren't getting pulled into this project. Personally, I'm leaning towards https://hub.docker.com/r/romancin/rutorrent/ it seems to have lots of extra goodies baked in.
